tests/eval_hub/evalhub_kueue_integration/utils.py

```
@@ -0,0 +1,194 @@
+        payload["queue"] = {"kind": "kueue", "name": queue_name}
+
+    if priority is not None:
+        payload["priority"] = priority
+
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Content-Type": "application/json",
+        "X-Tenant": tenant,
+    }
+
+    resp = requests.post(f"{base_url}{EVALHUB_JOBS_ENDPOINT}", json=payload, headers=headers, verify=False, timeout=30)
```

Ruff (0.15.12)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)